app/src/main/java/org/fossasia/phimpme/accounts/CloudRailServices.java

```diff
@@ -27,6 +28,7 @@
     private final AtomicReference<CloudStorage> dropbox = new AtomicReference<>();
     private Activity context = null;
     DropboxLogin dropboxLogin;
+    GoogleDrive googleDrive;
```

Comment:
Codacy says this should be `private` I think.

Reply:
@mayank8318 The codeacy changes has to be reverted because travis started failing, and I think we can ignore codeacy for this change.

@anantprsd5 Your reviews?